pkg/fakedata/generator.go

```diff
@@ -191,7 +191,7 @@ func init() {
 	}
 
 	generators["domain.name"] = Generator{
-		Name: "domain.tld",
+		Name: "domain.name",
```

Comment:
the integration tests caught this :) One of the tests would randomly fail because two generators had the same name. Funny finding